app/assets/javascripts/components/books/details.jsx

```diff
@@ -0,0 +1,59 @@
+class BooksDetails extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      book: {},
+    };
+  }
+
+  componentDidMount() {
+    this.getData();
+  }
+
+  getData() {
+    $.ajax({
```

Comment:
@danTeteruk Can we put all the ajax requests to one separate file (it's a lot easier to have all the API calls in one place) and instead of using jQuery ajax lets try to make native API calls via Axios or Superagent etc.

Reply:
good idea, thanks, I'll try to do it now